src/Components/FeatherIcons.re

```diff
@@ -0,0 +1,148 @@
+open Revery;
+open Revery.UI.Components;
+
+let colorToCSS = color => {
+  let (r, g, b, _) = Color.toRgba(color);
+  Printf.sprintf(
+    "#%02X%02X%02X",
+    int_of_float(r *. 255.),
+    int_of_float(g *. 255.),
+    int_of_float(b *. 255.),
```

Comment:
Can we include alpha here? I think some themes leverage it

Reply:
Unfortunately the SVG backend doesn't seem to like it when you specify an alpha value, either with 8 digit hex or `rgba(...)`. It just doesn't render the stroke, so nothing appears.

Reply:
Ah, gotcha - thanks for trying out!